mesheryctl/internal/cli/root/config/config.go

```diff
@@ -231,6 +231,75 @@ func AddTokenToConfig(token Token, configPath string) error {
 	return nil
 }
 
+// DeleteTokenFromConfig deletes a token passed to it to mesheryctl config file
+func DeleteTokenFromConfig(token Token, configPath string) error {
+	if _, err := os.Stat(configPath); os.IsNotExist(err) {
+		return err
+	}
+
+	viper.SetConfigFile(configPath)
+	err := viper.ReadInConfig()
+	if err != nil {
+		return err
+	}
+
+	mctlCfg, err := GetMesheryCtl(viper.GetViper())
+	if err != nil {
+		return errors.Wrap(err, "error processing config")
```

Comment:
Let's use errors.new() - MeshKit style errors, yeah? 

@Shreyas220, be a stickler about this, please.